### ISSUE TITLE
[Accessibility- Script C 7.1] Lack of some labels in App center

### ISSUE THREAD

**Issue by mbensalem** [Meeds-io/meeds#3839]
### Problem Statement

Environment: 7.x

#### Steps to Reproduce

- Access to App center in extended mode

- Add a category to an application

- Activate the filter by category

#### Current behavior:

- The filter of application doesn't have a label

- The Home button (after activating category) doesn't have a label

#### Expected behavior:

- Add an aria-label="Filter applications" to the select of filter

- Add an aria-label="Home applications" to the Home button

**Comment by mbensalem** [comment]
Still reproduced on 7.2.x
